motor_control/Triple_HBridge/ramping.ino

```diff
@@ -0,0 +1,67 @@
+#include "ramping.h"
+
+Ramping::Ramping():m_throttleIP(0), m_currentSpd(0), m_desiredSpd(0){}
+void Ramping::setThrottle(int throttle){ m_throttleIP = throttle; }
+void Ramping::setCurrentSpd(byte spd)      {  m_currentSpd = spd; }
+void Ramping::setDesiredSpd(byte spd)      {  m_desiredSpd = spd; }
+
+int  Ramping::getThrottle()   const { return m_throttleIP;  }
+byte Ramping::getCurrentSpd() const {  return m_currentSpd; }
+byte Ramping::getDesiredSpd() const {  return m_desiredSpd; }
+
+void Ramping::spdUp(byte amount)   {  m_currentSpd += amount; }
```

Comment:
is it possible for these byte variables to overflow?

Reply:
I would add the if statements for variables not to overflow and modify newSpd method related to that